lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -280,7 +280,7 @@ def classification
 
             if can_forward_all?
               :all
-            elsif target_ruby_version >= 3.2
```

Comment:
We check this condition again on line 105 (but by then we would have already rejected the false-positive this PR fixes)